src/main.rs

```diff
@@ -139,9 +147,13 @@ fn main() {
     // transfer project to build server
     let mut rsync_to = Command::new("rsync");
     rsync_to.arg("-a".to_owned())
-        .arg("--delete")
-        .arg("--info=progress2")
-        .arg("--exclude")
+        .arg("--delete");
+
+    if progress {
+        rsync_to.arg("--info=progress2");
```

Comment:
this is called `--progress` in `rsync` options on MacOS. So the function should check the OS before applying this feature.

Reply:
Thank you for the hint. Can you check if this branch works for you on Mac OS now? Then I'd merge it.